lte/gateway/c/core/BUILD.bazel

```diff
@@ -0,0 +1,1163 @@
+        "-DCMAKE_BUILD_TYPE=\\\"CMAKE_BUILD_TYPE\\\"",  # TODO(#11591): needs deprecation
+        "-DS1AP_DEBUG_LIST=false",  # TODO: Determine if this is even used / merits commit to code base
+        "-DSCTP_DUMP_LIST=false",  # TODO: Determine if this is even used / merits commit to code base
+        "-DTRACE_HASHTABLE=false",  # TODO: Determine if this is even used / merits commit to code base
+        "-DTRACE_3GPP_SPEC=false",  # TODO: Determine if this is even used / merits commit to code base
+    ],
+    deps = [
+        ":asn1_r15",
+        ":asn1_r16",
+        ":sentry_log",
+        "//feg/protos:csfb_cpp_grpc",
+        "//feg/protos:s6a_proxy_cpp_grpc",
+        "//feg/protos:s6a_proxy_cpp_proto",
+        "//feg/protos:s8_proxy_cpp_grpc",
+        "//lte/gateway/c/core/oai/common/glogwrapper:glog_logging",
+        "//lte/gateway/c/core/oai/lib/bstr:bstrlib",
+        "//lte/gateway/c/core/oai/lib/directoryd:directoryd_client",
+        "//lte/gateway/c/core/oai/lib/event_client:eventd_client",
+        "//lte/protos:ha_orc8r_cpp_grpc",
+        "//lte/protos:ha_service_cpp_grpc",
+        "//lte/protos:mconfigs_cpp_proto",
+        "//lte/protos:mobilityd_cpp_grpc",
+        "//lte/protos:pipelined_cpp_grpc",
+        "//lte/protos:s1ap_service_cpp_grpc",
+        "//lte/protos:s6a_service_cpp_grpc",
+        "//lte/protos:s6a_service_cpp_proto",
+        "//lte/protos:sctpd_cpp_grpc",
+        "//lte/protos:session_manager_cpp_grpc",
+        "//lte/protos:sms_orc8r_cpp_grpc",
+        "//lte/protos:spgw_service_cpp_grpc",
+        "//lte/protos:spgw_service_cpp_proto",
+        "//lte/protos:subscriberauth_cpp_grpc",
+        "//lte/protos:subscriberdb_cpp_grpc",
+        "//lte/protos/oai:common_types_cpp_proto",
+        "//lte/protos/oai:mme_nas_state_cpp_proto",
+        "//lte/protos/oai:ngap_state_cpp_proto",
+        "//lte/protos/oai:s1ap_state_cpp_proto",
+        "//lte/protos/oai:sgw_state_cpp_proto",
+        "//orc8r/gateway/c/common/async_grpc:async_grpc_receiver",
+        "//orc8r/gateway/c/common/config:mconfig_loader",
+        "//orc8r/gateway/c/common/ebpf",
+        "//orc8r/gateway/c/common/logging",
+        "//orc8r/gateway/c/common/sentry:sentry_wrapper",
+        "//orc8r/gateway/c/common/service303",
+        "//orc8r/protos:eventd_cpp_proto",
+        "//orc8r/protos:mconfigs_cpp_proto",
+        "//orc8r/protos:redis_cpp_proto",
+        "@cpp_redis",
+        "@libfluid_base//:fluid_base",
+        "@libfluid_msg//:fluid_msg",
+        "@liblfds//:lfds710",
+        "@system_libraries//:czmq",
+        "@system_libraries//:folly",
+        "@system_libraries//:libconfig",
+        "@system_libraries//:libfd",
+        "@system_libraries//:libgnutls",
+        "@system_libraries//:libnettle",
+        "@system_libraries//:sctp",
+    ],
+)
+
+cc_binary(
+    name = "oai_mme",
+    srcs = [
+        "oai/oai_mme/oai_mme.c",
+    ],
+    deps = [
+        ":core",
+    ],
+)
+
+cc_library(
```

Comment:
Why not in `oai/common/BUILD.bazel`?

Reply:
Yes - you got it. Per discussion in #11892 we cannot have a module in oai/common/ unless all files can be in oai/common/ and unfortunately there are cycles with some components in oai/common that require inclusion in `//lte/gateway/c/oai/core:core` today.  What I would advise (and we could do this as soon as Bazelification is complete and CMake is turned down) is to move `oai/common/sentry_log.{h,cpp}` to a new directory and then give it a module in that directory to untangle this issue.